apps/escuelafrontend/src/pages/instructores/[slug].tsx

```diff
@@ -0,0 +1,85 @@
+import * as React from 'react'
+import groq from 'groq'
+import {sanityClient} from 'utils/sanity-client'
+import ProfileTemplate from 'layouts/profile'
+
+const InstructorProfile = (props: any) => {
+  return <ProfileTemplate meta={props.instructorResource[0]} />
+}
+
+const allInstructorsQuery = groq`*[_type == "person"]{
+  "slug": slug.current
+}
+`
+
+const allResourcesQuery = groq`*[_type == "person" && slug.current == $slug][0]{
+  "slug": slug.current,
+  "instructorResource": *[_type=='collaborator' && references(^._id)]{ 
+    "instructor": {
+      'slug': person->slug.current,
+      'name': person->name,
+      'twitter': person->twitter,
+      'website': person->website,
+      'image': person->image.url,
+      'bio': person->bio,
+      'title': person->title,
+      "featuredResource": person->featuredResource[]-> {
+        title,
+        slug,
+        path,
+        publishedAt,
+        "tag": softwarelibraries[0]-> {
+          name,
+          "image": image.url
+        },
+      }
+    },
+    "allArticles": *[_type=='post' && references(^._id)]{ 
+      title,
+      slug,
+      path,
+      publishedAt,
+      "tag": softwarelibraries[0]-> {
+        name,
+        "image": image.url
+      },
+      "instructor": collaborators[0]->{
+        'slug': person->slug.current,
+        'name': person->name,
+        'twitter': person->twitter,
+        'image': person->image.url
+      },
+    }
```

Comment:
`instructor ` pulls all the data from Sanity to build the portfolio 

`allArticles` pulls the articles related to instructor via a `references(^._id)`